### PR TITLE
GEODE-8095: Changes to make GEODE Respond to Restore Redundancy REST Command

### PR DESCRIPTION
**Pull request by mhansonp**
These are the core changes that will allow the invocation of Restore Redundancy from REST

### CODE REVIEW

File: geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RestoreRedundancyOperationController.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        Optional.ofNullable(securityService).map(SecurityService::getSubject).map(Object::toString)
+            .orElse(null));
+    ClusterManagementOperationResult<RestoreRedundancyRequest, RestoreRedundancyResults> result =
+        clusterManagementService
+            .start(operation);
+    return new ResponseEntity<>(result, HttpStatus.ACCEPTED);
+  }
+
+  @ApiOperation(value = "list restore-redundancy")
+  @PreAuthorize("@securityService.authorize('DATA', 'MANAGE')")
```
Comment (jinmeiliao) [inline geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RestoreRedundancyOperationController.java:59]: Is this the right authorization code we need to  use?

File: geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RestoreRedundancyOperationController.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  @GetMapping(RESTORE_REDUNDANCY_ENDPOINT)
+  public ClusterManagementListOperationsResult<RestoreRedundancyRequest, RestoreRedundancyResults> listRestoreRedundancies() {
+    return clusterManagementService.list(new RestoreRedundancyRequest());
+  }
+
+  @ApiOperation(value = "get restore-redundancy")
+  @PreAuthorize("@securityService.authorize('DATA', 'MANAGE')")
```
Comment (jinmeiliao) [inline geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RestoreRedundancyOperationController.java:66]: check this as well

File: geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RestoreRedundancyOperationController.java (hunk 1 above)
Comment (mhansonp) [reply geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RestoreRedundancyOperationController.java:59]: How do I check it?

Comment (mhansonp) [reply geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RestoreRedundancyOperationController.java:59]: We are agreed that this is correct.

File: geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RestoreRedundancyOperationController.java (hunk 2 above)
Comment (mhansonp) [reply geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RestoreRedundancyOperationController.java:66]: We are agreed that this is correct.

**Pull request merged**